apluslms_roman/cli.py

```diff
@@ -365,20 +435,155 @@ def build_action(context):
     return result.code
 
 
+def init_action(context):
+    cwd = getcwd()
+    try:
+        config = ProjectConfig.find_from(cwd)
+        exit(1, _("A project configuration already exists at {}".format(config.path)))
+    except FileNotFoundError:
+        pass
+
+    if context.args.project_config:
+        file_ = context.args.project_config
+        if '.' not in file_:
+            is_recognized = False
+        else:
+            name, prefix = file_.split('.', 1)
+            if (name not in ProjectConfig.DEFAULT_NAMES
+                    or prefix not in ProjectConfig.DEFAULT_PREFIXES):
+                is_recognized = False
+
+        if not is_recognized:
+            warning(_(
+                "roman won't recognize {} as a project config file without the -f flag"
+            ).format(file_))
+    else:
+        file_ = ProjectConfig.DEFAULT_FILENAME
+    ProjectConfig.load(path_join(cwd, file_), allow_missing=True).save()
+    print(_("Project configuration file {} created successfully.".format(file_)))
+
+
 def config_print_action(context):
     all_ = not any(getattr(context.args, k, False) for k in ('user', 'project'))
     if context.args.debug:
         print("---\n# arguments:")
         data = {k: v for k, v in vars(context.args).items() if k[0] != '_' and not callable(v)}
         yaml_dump(data, stdout)
-    if all_ or context.args.user:
+    if all_ or context.args.global_:
         print("---\n# roman settings:")
         yaml_dump(context.settings._data, stdout)
     if all_ or context.args.project:
         print("---\n# project config")
         yaml_dump(get_config(context)._data, stdout)
 
 
+def config_set_action(context):
+    if context.args.global_ == context.args.project:
+        exit(1, "Choose either global settings or project settings")
```

Comment:
If neither are set, then default to project config, so `roman config set foo=bar` uses project config. If user would say `roman config -g -p` then we need to raise error. `-p` is mostly for someone to be explicit (e.g. to ensure we write project config in a script).